src/xtgeo/grid3d/grid.py

```diff
@@ -2422,7 +2414,7 @@ def reverse_row_axis(
         _grid_etc1.reverse_row_axis(self, ijk_handedness=ijk_handedness)
         self._tmp = {}
 
-    def make_zconsistent(self, zsep: float = 1e-5) -> None:
+    def make_zconsistent(self, zsep: float | int = 1e-5) -> None:
```

Comment:
The ` | int` is unnecessary?

Reply:
The method inside _grid_etc1 that this one is calling has specific handling of int input. Hence I changed it.
https://github.com/equinor/xtgeo/blob/56d76c34c29f2892623c880c8bac42c4d82a6677/src/xtgeo/grid3d/_grid_etc1.py#L887-L895